.github/workflows/api_test.yaml

```
@@ -0,0 +1,24 @@
+    runs-on: ubuntu-latest
+    name: API Tests
+    services:
+      api:
+        image: ghcr.io/${{ github.repository }}:latest
+        env:
+          GRADIO_SERVER_PORT: 7860
+          GRADIO_SERVER_NAME: 0.0.0.0
+        ports:
+          - 7860:7860
+    steps:
+      - name: Checkout 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)